.circleci/config.yml

```diff
@@ -164,6 +174,25 @@ jobs:
       - npm_install_from_cache
       - build_dashboard
 
+  release_storybook:
+    docker:
+      - image: circleci/node:12.18.0-stretch
+    steps:
+      - git_checkout_from_cache
+      - npm_install_from_cache
+      - build_storybook
+      - versioning:
+          target_branch: storybook
```

Comment:
i didn't find any `storybook` branch in this repository yet

Reply:
Yes, that variable had misleading name. I changed it to `version_name` for readability.